dashboard/test/ui/features/support/blockly_helpers.rb

```diff
@@ -24,6 +24,7 @@ def generate_begin_to_drag_code(from, to, target_dx, target_dy)
         "$(\"[#{id_selector}='#{from}']\").simulate( 'drag', {justDrag: true, handle: 'corner', dx: drag_dx + #{target_dx}, dy: drag_dy + #{target_dy}, moves: 5});"
   end
 
+  # To-Do: Combine these next two:
```

Comment:
super nit: use "TODO" to make searching the repo for TODOs consistent

Reply:
Noted, but this comment was meant to be removed during clean-up. Deleted!